scripts/lua/sidebar.lp

```diff
@@ -43,35 +43,43 @@
                 <!-- Group Management -->
                 <li class="menu-group<? if scriptname == 'groups' then ?> active<? end ?>">
                     <a href="<?=webhome?>groups">
-                        <i class="fa fa-fw menu-icon fa-user-friends"></i> <span>Groups</span>
-                        <span class="pull-right-container">
-                            <span class="label label-primary pull-right" id="num_groups" title="Number of enabled groups"></span>
+                        <i class="fa fa-fw menu-icon fa-user-friends"></i>
```

Comment:
Why these changes? Is the resulting HTML still valid?

Reply:
NVM it's all good 👍

Reply:
Yes, it is valid HTML.

You identified there were missing `</span>` closing tags and added the closing tags, but the theme expects the `<span class="pull-right-container">` to be inside the other `<span>`.